docs/modules/ROOT/pages/index.adoc

```diff
@@ -30,7 +30,7 @@ Rather, it's a tool for creating highly aesthetic, professional, _easy-to-read_
 Let's face it, many of the technical e-books out there--especially those produced from software documentation--are *_hideous_*.
 {project-name} is on a mission to disrupt the status quo.
 
-.An excerpt from an e-book produced by {project-name} shown in Day, Night and Sepia mode.
+.An excerpt from an e-book produced by {project-name} shown in Day, Night, and Sepia modes.
```

Comment:
I'm not sure why you want to put a comma here

Reply:
I am very glad to make such minor corrections. As far as the comma placement, the technical term for it is the "Oxford comma," which grammatical idiom makes it explicit, for example in this case, that "Night" and "Sepia" are two different modes, rather than allowing the ambiguity that "Night and Sepia" is the name of one mode. As this Oxford comma idiom was already in use elsewhere in the document, I merely proposed this change for the sake of consistency.

Reply:
> "Oxford comma,"

Cool, today I've learned something new :D